torch/_inductor/codegen/cpp_prefix.h

```diff
@@ -886,6 +886,105 @@ void mm_get_cache_blocking(
   }
 }
 
+// Transpose a [2, 32] matrix to [32, 2]
+// Note: the output leading dimension should be 2,
+// that is, the output must be contiguous
+template <typename scalar_t, typename = std::enable_if_t<sizeof(scalar_t) == 2>>
+static inline void transpose_pad_2x32_block(
```

Comment:
Why do we want to complicate the `cpp_prefix.h` here? Can we move them to aten so that the same function can be shared by both eager kernel and compiler generated code?